pkg/onit/cluster/service.go

```diff
@@ -46,14 +49,61 @@ type Port struct {
 // Service is the base type for multi-node services
 type Service struct {
 	*Deployment
-	ports      []Port
-	replicas   int
-	debug      bool
-	user       *int
-	privileged bool
-	secrets    map[string]string
-	env        map[string]string
-	args       []string
+	ports         []Port
+	replicas      int
+	debug         bool
+	user          *int
+	privileged    bool
+	secrets       map[string]string
+	env           map[string]string
+	args          []string
+	cpuRequest    string
+	memoryRequest string
+	cpuLimit      string
+	memoryLimit   string
+}
+
+// CPURequest returns the cpu request for a deployment container
+func (s *Service) CPURequest() string {
+	return GetArg(s.name, "cpu.request").String(s.cpuRequest)
```

Comment:
`GetArg` is a variadic function that puts `.` between the arguments, so the correct call would be `GetArg(s.name, "cpu", "request")`

Reply:
Good to know. Fixed.